src/main/config/config.c

```diff
@@ -672,7 +672,11 @@ void validateAndFixGyroConfig(void)
         /* If bidirectional DSHOT is being used on an F4 or G4 then force DSHOT300. The motor update restrictions then applied
          * will automatically consider the loop time and adjust pid_process_denom appropriately
          */
+#ifdef USE_PID_DENOM_OVERCLOCK_LEVEL
+        if ((systemConfig()->cpu_overclock < USE_PID_DENOM_OVERCLOCK_LEVEL) &&  (motorConfig()->dev.useDshotTelemetry)) {
+#else
         if (motorConfig()->dev.useDshotTelemetry) {
+#endif
```

Comment:
Personally, I'd do it this way, yours is fine
```suggestion
        if (true
#ifdef USE_PID_DENOM_OVERCLOCK_LEVEL
        && (systemConfig()->cpu_overclock < USE_PID_DENOM_OVERCLOCK_LEVEL) 
#endif
        && motorConfig()->dev.useDshotTelemetry
        ) {
```